sessionctx/variable/tidb_vars.go

```diff
@@ -78,6 +78,9 @@ const (
 	// If we can't estimate the size of one side of join child, we will check if its row number exceeds this limitation.
 	TiDBBCJThresholdCount = "tidb_broadcast_join_threshold_count"
 
+	// TiDBBCJoinCostModelVersion indicates the version of cost model used for mpp broadcast join
+	TiDBBCJoinCostModelVersion = "tidb_broadcast_join_cost_model_version"
```

Comment:
Can we rename it to `tidb_prefer_broadcast_join_xxx` to avoid the keyword `cost_model`?
All code about the cost model should be in `plan_cost_ver2.go`.
And strictly speaking, the logic here is rule-based instead of cost-based (force MPP if the exchanging data size is smaller than ShuffleJoin here).

Reply:
Rest LGTM

Reply:
Done. Renamed to `tidb_broadcast_join_cost_model_version`